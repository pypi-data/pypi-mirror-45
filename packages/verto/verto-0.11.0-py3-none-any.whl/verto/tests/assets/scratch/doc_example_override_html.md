```scratch
when flag clicked
say [Hi]
```
