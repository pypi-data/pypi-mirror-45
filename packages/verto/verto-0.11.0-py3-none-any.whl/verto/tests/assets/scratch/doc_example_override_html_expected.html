<div class="scratch-inline-images">
<img class="scratch-blocks" onerror="this.src='scratch-blocks-a3b77ed3c3fa57e43c830e338dc39d292c7def676e0e8f7545972b7da20275da.png'" src="scratch-blocks-a3b77ed3c3fa57e43c830e338dc39d292c7def676e0e8f7545972b7da20275da.svg">
</div>
